Core/HLE/sceIo.cpp

```diff
@@ -557,7 +558,6 @@ static void __IoAsyncEndCallback(SceUID threadID, SceUID prevCallbackId) {
 	}
 }
 
-static DirectoryFileSystem *memstickSystem = nullptr;
 static DirectoryFileSystem *exdataSystem = nullptr;
 #if defined(USING_WIN_UI) || defined(APPLE)
 static DirectoryFileSystem *flash0System = nullptr;
```

Comment:
Oops, these should be removed too right?

-[Unknown]

Reply:
Uh, yeah. Oops indeed